packages/js/tests/components/WincherKeyphrasesTable.test.js

```diff
@@ -12,7 +12,7 @@ import { trackKeyphrases } from "../../src/helpers/wincherEndpoints";
 
 jest.mock( "../../src/helpers/wincherEndpoints" );
 trackKeyphrases.mockImplementation( async fn => {
-	await fn();
+	return fn;
```

Comment:
This change is not needed for this PR but fixes an error in one of the other unit tests in this file. So I thought I might as well fix it.